roles/kafka-ws/templates/kafkaws_deployment.yaml.j2

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    name: kafkaws
+    app: dojot
+  name: kafkaws
+  namespace: {{ dojot_namespace }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: kafkaws
+        app: dojot
+    spec:
+      containers:
+      - image: dojot/kafka-ws:{{ dojot_kafkaws_version }}
+        name: kafkaws
+        env:
```

Comment:
The new Kafka WS version has changed some environment variables, we need to double check this section.

Reply:
`KAFKA_WS_JWT_HEADER_AUTH` it no longer exists, the idea is to control access through the _accessed route_, in this case `/<base_url>/v1/topics/*` requires a ticket and `/<base_url>/v1/public/topics/*` is for public use, but `/<base_url>/v1/public/topics/*` does not yet exist, it would have to be mapped if necessary.